cg/meta/report/report_api.py

```diff
@@ -103,8 +103,18 @@ def get_delivery_report_from_hk(self, case_id: str, version: Version) -> str | N
         return delivery_report.full_path
 
     def get_scout_uploaded_file_from_hk(self, case_id: str, scout_tag: str) -> str | None:
-        """Return the file path of the uploaded to Scout file given its tag."""
-        raise NotImplementedError
+        """Return file path of the uploaded to Scout file given its tag."""
+        version: Version = self.housekeeper_api.last_version(bundle=case_id)
+        tags: list = self.get_hk_scout_file_tags(scout_tag=scout_tag)
+        uploaded_file: File = self.housekeeper_api.get_latest_file(
+            bundle=case_id, tags=tags, version=version.id
+        )
+        if not tags or not uploaded_file:
+            LOG.warning(
```

Comment:
should this only be a warning or raise an error?

Reply:
It was as warning so we don't want to change the behaviour here

Reply:
It should only be a warning.  The delivery report has its own validation of these fields and if mandatory it throws an error